components/pickerbutton/stories/template.js

```diff
@@ -70,3 +71,66 @@ export const Template = ({
 		</button>
 	`;
 };
+
+/**
+ * Displays the component with a custom icon (instead of the chevron UI icon).
+ * Two examples are shown; with a custom UI icon and a custom Workflow icon.
+ */
+export const CustomIconTemplate = (args) => html`
```

Comment:
Ideally I'd like to move away from hardcoding layouts and headings in the templates for the docs pages but I think this is an edge case where it makes sense for now. I'd love to brainstorm if there are ways we can support these custom demos without a lot of custom grids and headings.